enterprise_subsidy/apps/api/v1/serializers.py

```diff
@@ -306,6 +306,30 @@ def create(self, validated_data):
         return transaction
 
 
+# pylint: disable=abstract-method
+class SubsidyLearnerAggregateSerializer(serializers.Serializer):
```

Comment:
[question] Why are we using `serializers.Serializer` vs `serializers.ModelSerializer` like the examples above?

Reply:
`ModelSerializer` is a convenience class for more easily serializing a Django model.  Since there's no model related to this flow (computing aggregates by learner), we can't use `ModelSerializer`.
https://www.django-rest-framework.org/api-guide/serializers/#modelserializer

Reply:
+1 to what @iloveagent57 has to say- while we query the subsidy model, we're not directly returning the records but instead modifying and calculating an aggregate from the data. Not returning the records from the model means that a model serializer wouldn't work for us